recipe/meta.yaml

```diff
@@ -46,7 +46,7 @@ source:
 build:
   number: 1
   # Bug in for cuda 11.1 with c++17 on linux: https://forums.developer.nvidia.com/t/nvc-20-9-fails-to-compile-code-instantiating-any-std-tuple-with-gcc-10-2-on-c-17/160987
-  skip: true  # [cuda_compiler_version == "10.2" or (linux and cuda_compiler_version == "11.1")]
+  skip: true  # [cuda_compiler_version == "12.0" or cuda_compiler_version == "10.2" or (linux and cuda_compiler_version == "11.1")]
```

Comment:
Anything below 11.2 has been droppped from conda-forge, so we can simplify this condition (and remove the comment above)